lib/src/client_dart.dart

```diff
@@ -109,6 +67,19 @@ class HomeConnectApi {
     await storage.setCredentials(tokens);
   }
 
+  Future<http.Response> put(
```

Comment:
we should add the same refresh token logic here

Reply:
done :+1: